benchmarks/benchmarks/scattering2d.py

```diff
@@ -1,5 +1,5 @@
-import torch
-import kymatio.scattering2d.backend as backend
+#import torch
+#import kymatio.scattering2d.backend as backend
```

Comment:
from kymatio.numpy import Scattering2D

Reply:
why are these lines now commented?
Shouldn't they either be in code or deleted?